### PR TITLE
Show JoinEUI and DevEUI for activation in live traffic view

### LINKED ISSUE #6981: Show JoinEUI and DevEUI for activation in live traffic view
### Summary

Show the JoinEUI And DevEUI for activation in live traffic view.

### Current Situation

Currently, we show the (new) DevAddr. The EUIs are only shown for some events:

xBAF2OsCjWmO2PUD

### Why do we need this? Who uses it, and when?

During activation, what's more relatable are the EUIs as they are stable. The new DevAddr is useful too, but not as useful as it is presented now as the only identifier.

### Proposed Implementation

I think this is frontend only as these identifiers should be available in the events already.

### Contributing

- [X] I can help by doing more research.
- [ ] I can help by implementing the feature after the proposal above is approved.
- [X] I can help by testing the feature before it's released.

### Code of Conduct

- [X] I agree to follow TTN's [Community Code of Conduct](https://github.com/TheThingsNetwork/Manifesto/blob/master/CodeOfConduct.md).

### PR DESCRIPTION
**Pull request by PavelJankoski**
#### Summary

Closes #6981 

#### Changes

- Added columns JoinEUI and DevEUI in live data views.

image

#### Testing

1. Login to console.
2. Go to applications.
3. Select some active application.
4. Click on Live data.
5. Check the rows if joinEUI and devEUI columns are present.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] The steps/process to test this feature are clearly explained including testing for regressions.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by KrishnaIyer** [comment]
@PavelJankoski: Could you please add some screenshots?

File: pkg/webui/console/components/events/previews/application-uplink.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ const ApplicationUplinkPreview = React.memo(({ event }) => {
   return (
     <DescriptionList>
       <DescriptionList.Byte title={messages.devAddr} data={deviceIds.dev_addr} />
+      <DescriptionList.Byte title={sharedMessages.joinEUI} data={deviceIds.join_eui} />
+      <DescriptionList.Byte title={sharedMessages.devEUI} data={deviceIds.dev_eui} />
```
Comment (kschiffer) [inline pkg/webui/console/components/events/previews/application-uplink.js]:
```suggestion
```
Let's use this JoinEUI here only, to give more space to the payload.

File: pkg/webui/console/components/events/previews/application-uplink-normalized.js [hunk 2]
```diff
@@ -29,6 +29,8 @@ const ApplicationUplinkNormalizedPreview = React.memo(({ event }) => {
   return (
     <DescriptionList>
       <DescriptionList.Byte title={messages.devAddr} data={deviceIds.dev_addr} />
+      <DescriptionList.Byte title={sharedMessages.joinEUI} data={deviceIds.join_eui} />
+      <DescriptionList.Byte title={sharedMessages.devEUI} data={deviceIds.dev_eui} />
```
Comment (kschiffer) [inline pkg/webui/console/components/events/previews/application-uplink-normalized.js]:
```suggestion
```

File: pkg/webui/console/components/events/previews/application-downlink.js [hunk 3]
```diff
@@ -29,6 +29,8 @@ const ApplicationDownlinkPreview = React.memo(({ event }) => {
   return (
     <DescriptionList>
       <DescriptionList.Byte title={messages.devAddr} data={deviceIds.dev_addr} />
+      <DescriptionList.Byte title={sharedMessages.joinEUI} data={deviceIds.join_eui} />
+      <DescriptionList.Byte title={sharedMessages.devEUI} data={deviceIds.dev_eui} />
```
Comment (kschiffer) [inline pkg/webui/console/components/events/previews/application-downlink.js]:
```suggestion
```

**Review by kschiffer — APPROVED** [review]
Please resolve my comments before merging.

**Pull request merged**